ui/src/main.tsx

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
 import CssBaseline from "@mui/material/CssBaseline";
+import "./css/App.css"
```

Comment:
I would move the `App.css` to `NginxInstance.css` and put it in the `instances` directory. In this way the style sticks to the Component.